### DISCUSSION TITLE
reddit style news feed to keep track of latest updates

### DISCUSSION THREAD

**Discussion by fightOrF** [WeAreMahsaAmini/Core#172] [Ideas]
we can use an open source project like https://lobste.rs/ (source https://github.com/lobsters/lobsters ) to let people vote the latest news from iran and hopefully create a near real-time aggregated view of the news.

**Comment by ghost** [comment]
Why not just create a subreddit?

**Comment by fightOrF** [comment]
It would be lighter and easier to use with low bandwidth internet. We could also provide mirrors to circumvent censorship.

**Reply by ghost** [reply]
Reddit is pretty lightweight, and we can provide mirrors for subreddits. I think your idea is a very good one; the reason I insist is because creating a subreddit is much easier and faster. It is also useful for Iranians abroad and non-Iranians.

**Reply by fightOrF** [reply]
I think you are right. furthermore it would be hard to prove that such a website is not hosted by a hostile entity.

**Reply by fightOrF** [reply]
Thanks for feedback
